jina/flow/base.py

```diff
@@ -967,3 +972,15 @@ def identity(self, value: str):
 
     # for backward support
     join = needs
+
+    def rolling_update(self, pod_name):
+        """
+        Update pods one after another - only used for compound pods.
+
+        :param pod_name: pod to update
+        """
+        for i, replica in enumerate(self._pod_nodes[pod_name].replica_list):
+            replica.close()
```

Comment:
what about removing a `Pod` and restarting a new one with the same parameters? This way we have a more `functional` style and cleaner implementations?

Reply:
Sounds good - done